core/src/main/tut/README.md

```diff
@@ -189,11 +189,14 @@ A non-comprehensive list of other libraries which have integrated with PureConfi
 Apache Spark (specifically version 2.1.0) has a transitive dependency
 on [Shapeless](https://github.com/milessabin/shapeless) 2.0.0. This version is
 too old to be used by PureConfig, making your Spark project fail when using
-`spark-submit`.
+`spark-submit`. The solution is to shade, i.e. rename, the version of Shapeless
+used by PureConfig.
+
+#### SBT
 
 If you are using the [sbt-assembly](https://github.com/sbt/sbt-assembly) plugin
 to create your JARs you
-can [shade this dependency](https://github.com/sbt/sbt-assembly#shading) by
+can [shade](https://github.com/sbt/sbt-assembly#shading) shapeless by
```

Comment:
Nitpick++: can you standardize at "shapeless", since the author [always writes it lowercase](https://github.com/milessabin/shapeless), even in titles and in the beginning of sentences? :)